dev/TabView/TabViewItem.cpp

```diff
@@ -489,6 +489,35 @@ void TabViewItem::OnPointerCaptureLost(winrt::PointerRoutedEventArgs const& args
     RestoreLeftAdjacentTabSeparatorVisibility();
 }
 
+// Note that the ItemsView will handle the left and right arrow keys if we don't do so before it does,
+// so this needs to be handled below the items view. That's why we can't put this in TabView's OnKeyDown.
+void TabViewItem::OnKeyDown(winrt::KeyRoutedEventArgs const& args)
+{
```

Comment:
Would it work to just set the TabView tab strip to use XY focus?
Or does that not work because you only want left and right but not up and down?

Reply:
That, plus XY focus doesn't allow wrapping.